library/std/src/collections/hash/set.rs

```diff
@@ -588,6 +588,11 @@ where
     /// Visits the values representing the intersection,
     /// i.e., the values that are both in `self` and `other`.
     ///
+    /// Note: When an equal element is present in `self` and `other`
```

Comment:
It's not clear to me what distinction is being made between a "note" vs "not a note". This just looks like ordinary documentation to me.
```suggestion
    /// When an equal element is present in `self` and `other`
```